### ISSUE TITLE
on press delay example, selecting with 2 fingers make the dragged item stuck on screen

### ISSUE THREAD

**Issue by iliran11** [clauderic/react-sortable-hoc#440]
How to reproduce: 

1. Go to [activate onPress example](http://clauderic.github.io/react-sortable-hoc/#/advanced/activate-onpress-200ms?_k=rxxwni)

2. try to drag with your 2 fingers.  the item is stuck.

![image](https://user-images.githubusercontent.com/24987188/47841551-ab2c4800-ddc2-11e8-80d5-a5c235ca3fc5.png)

**Comment by josepbidegain** [comment]
Hi! how are u? I have the same problem, do you found some solution?

**Comment by iliran11** [comment]
@josepbidegain, no. @clauderic , do you know this problem? is PR welcome in this case?

**Comment by ae861995** [comment]
Hi! Any solution for this? Having the same problem.

**Comment by GGantulga** [comment]
+1

**Comment by mwojslaw** [comment]
Hi, any news about this one guys ? Can we expect some actions here ?

**Comment by davidchappy** [comment]
Is this problem a manifestation of [this scenario](https://github.com/facebook/react/issues/13113)?

**Comment by davidchappy** [comment]
@clauderic, is above the best way for me to submit a pull request for this issue?
